src/main/java/org/sitenv/vocabularies/loader/code/CptLoader.java

```diff
@@ -0,0 +1,93 @@
+package org.sitenv.vocabularies.loader.code;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.StrBuilder;
+import org.apache.log4j.Logger;
+import org.sitenv.vocabularies.loader.BaseVocabularyLoader;
+import org.sitenv.vocabularies.loader.VocabularyLoader;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+/**
+ * Created by Brian on 2/7/2016.
+ */
+@Component(value = "CPT")
+public class CptLoader extends BaseVocabularyLoader implements VocabularyLoader {
+    private static Logger logger = Logger.getLogger(CptLoader.class);
+    private String oid;
+
+    public CptLoader() {
+        this.oid = CodeSystemOIDs.CPT4.codesystemOID();
+    }
+
+    public long load(List<File> filesToLoad, DataSource ds) {
+    	long n = 0;
+    	JdbcTemplate t = new JdbcTemplate(ds);
+        BufferedReader br = null;
+        FileReader fileReader = null;
+        String insertQueryPrefix = codeTableInsertSQLPrefix;
+        try {
+            StrBuilder insertQueryBuilder = new StrBuilder(codeTableInsertSQLPrefix);
```

Comment:
This Java class is very similar to Icd9BaseLoader.java, so I used the latter as a reference for this code review.  In the above line, is it better to change it to: StrBuilder insertQueryBuilder = new StrBuilder(insertQueryPrefix); ?